CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.3.14
+-----
+### Fixes
```

Comment:
I think this is not a fix, but more like an Enhancement

Reply:
Changed to `Enhancements`